app/api/utility.py

```diff
@@ -211,9 +213,11 @@ def create_query(
             {{
                 SELECT ?subject (count(distinct ?session) as ?num_sessions)
                 WHERE {{
-                    ?subject a nb:Subject;
-                            nb:hasSession ?session.
-                    ?session nb:hasAcquisition/nb:hasContrastType ?image_modal.
+                    ?subject a nb:Subject.
+                    OPTIONAL {{
+                        ?subject nb:hasSession ?session.
+                        ?session nb:hasAcquisition/nb:hasContrastType ?image_modal.
+                    }}
```

Comment:
nice, does that end up counting to zero if the OPTIONAL doesn't exist?

Reply:
indeed!